jax/third_party/README.md

```diff
@@ -0,0 +1,4 @@
+This sub-directory contains third-party code for which Google does not have
+copyright. Each sub-directory should correspond to a third-party library and
+must contain the appropriate LICENSE file. 
+See [instructions](https://g3doc.corp.google.com/company/teams/opensource/releasing/preparing.md?cl=head#third-party-components).
```

Comment:
This is an internal link, but happily Google's opensource documentation has itself been opensourced:
https://opensource.google/docs/releasing/preparing/#third-party-components